suite/tests/linux/sigaction.c

```diff
@@ -78,9 +78,13 @@ test_query(int sig)
     memset((void *)&old_act, 0xff, sizeof(old_act));
     rc = sigaction(sig, NULL, &old_act);
     assert(rc == 0 && old_act.sa_sigaction == first_act.sa_sigaction &&
-           /* The flags do not match due to SA_RESTORER. */
-           /* The rest of mask is uninit stack values from the libc wrapper. */
+    /* The flags do not match due to SA_RESTORER. */
+    /* The rest of mask is uninit stack values from the libc wrapper. */
+#if defined(MACOS)
+           *(int *)&old_act.sa_mask == *(int *)&first_act.sa_mask);
```

Comment:
A typedef up top (or maybe IF_MACOS_ELSE) might be more readable.

Reply:
Done.